### PR TITLE
Added rf_virtual_media.py tool for managing virtual media on a system

### PR DESCRIPTION
**Pull request by mraineri**
rf_virtual_media.py has three commands:
* info: Read and print all virtual media instances
* insert: Specify a URI for an image and perform an InsertMedia action (or PATCH) on an appropriate virtual media instance
* eject: Perform an EjectMedia action (or PATCH) on a specified virtual media instance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -453,6 +452,100 @@ It will then locate the event service and traverse the members of the event dest
...
+usage: rf_virtual_media.py eject [-h] --id ID
+
+required arguments
+  --id ID, -i ID  The identifier of the virtual media instance to eject
+
+optional arguments:
+  -h, --help      show this help message and exit
+```
+
+Example: `rf_event_service.py -u root -p root -r https://192.168.1.100 unsubscribe -id 1`
```
Comment (billdodd) [inline README.md:542]:
```suggestion
Example: `rf_event_service.py -u root -p root -r https://192.168.1.100 eject -id 1`
```

**Review by billdodd — COMMENTED** [review]
LGTM. Just saw one small copy-and-paste issue in the README.

File: README.md (hunk 1 above)
Comment (mraineri) [reply README.md:542]: Ick, I forgot to update that; there are some other things wrong with it too. Thanks for pointing that out.

**Comment by mraineri** [comment]
Approved 2/26

**Pull request merged**